CarpetLib/src/copy_3d.cc

```diff
@@ -116,21 +116,20 @@ void copy_3d(T const *restrict const src, ivect3 const &restrict srcpadext,
   ptrdiff_t const dstkoff = dstoff[2];
 
   // Loop over region
-  if (use_openmp) {
+  if (false and use_openmp) {
```

Comment:
this should not be committed in this form. Rather the whole "if" branch should be removed.